### PR TITLE
remove unused deprecated intrinsics `fptoui_auto`, `fptosi_auto`

### PR DESCRIPTION
**Pull request by JeffBezanson**

### CODE REVIEW

**Comment by tkelman** [comment]
Would anything outside of base have been using these? Not like they're particularly visible or documented, just wondering.

**Review by vtjnash — APPROVED** [review]

**Comment by JeffBezanson** [comment]
I think it's very unlikely there are any uses of this. If there are, it should be easy to update.

File: src/intrinsics.cpp [hunk 1]
```diff
@@ -833,36 +829,6 @@ static jl_cgval_t emit_intrinsic(intrinsic f, jl_value_t **args, size_t nargs,
...
-        if (!jl_is_bitstype(x.typ))
-            return emit_runtime_call(f, argv, nargs, ctx);
-        Type *xt = bitstype_to_llvm(x.typ);
-        Type *to = INTT(xt);
-        xt = FLOATT(xt);
-        if (!xt)
-            return emit_runtime_call(f, argv, nargs, ctx);
-        Value *from = emit_unbox(xt, x, x.typ);
-        Value *ans = builder.CreateFPToUI(from, to);
-        return mark_julia_type(ans, false, JL_JLUINTT(xt), ctx);
```
Comment (vtjnash) [inline src/intrinsics.cpp]: can delete the `JL_JLUINTT` helper function too

**Comment by StefanKarpinski** [comment]
Failure is unrelated: network failure during Pkg testing.

**Pull request merged**